db/migrate/20160406135108_create_flood_risk_engine_contacts.rb

```diff
@@ -17,5 +17,10 @@ def change
     add_foreign_key :flood_risk_engine_contacts,
                     :flood_risk_engine_organisations,
                     column: :partnership_organisation_id
+
+    # This fails in POSTGRES
```

Comment:
Do we at least need to flag this as a `# TODO: Fix this for postgres. Currently fails`